cobrand_hackney/static/js.js

```diff
@@ -37,3 +37,64 @@ document.querySelectorAll('.case-detail__map__overlay').forEach(function(b) {
 });
 
 })();
+
+// Creating a new dropdown "Case Locations"
+var caseLocationDiv = document.createElement('div');
+caseLocationDiv.className = 'govuk-form-group lbh-form-group';
+caseLocationDiv.id = 'div_id_case_location';
+caseLocationDiv.innerHTML = '<label for="id_case_location" class="govuk-label lbh-label">Case location</label>' + '<select class="govuk-select lbh-select" id="id_case_location"> <option value="all_areas" selected>All areas</option> <option value="selected_areas">Selected areas</option> <option value="outside_hackney">Outside Hackney</option> </select> ';
+
+// Defining the div that contains all the ward checkboxes
+var area = document.getElementById("div_id_ward");
+
+// Locating "Case Locations" before the area
+parentFormNode = area.parentNode
```

Comment:
You'll need semicolons at the end of all statements (here and line below, couple of other places too).